ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -11821,73 +11800,76 @@ void HqlCppTranslator::buildScriptFunctionDefinition(BuildCtx &funcctx, IHqlExpr
         HqlExprArray args;
         args.append(*LINK(ctxVar));
         IHqlExpression * param = formals->queryChild(i);
-        ITypeInfo *paramType = param->queryType();
-        IIdAtom * paramId = param->queryId();
-        const char * paramNameText = str(paramId);
-        if (!options.preserveCaseExternalParameter)
-            paramNameText = str(lower(paramId));
-        args.append(*createConstant(paramNameText));
-        IIdAtom * bindFunc;
-        switch (paramType->getTypeCode())
+        if (param != optionsParam)
```

Comment:
minor, could be moved earlier.  Possibly simpler as if (param == optionParam) continue;